ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -312,7 +312,12 @@ open class PillButtonBar: UIScrollView {
             let button = createButtonWithItem(item)
             buttons.append(button)
             stackView.addArrangedSubview(button)
-            button.accessibilityHint = String(format: "Accessibility.MSPillButtonBar.Hint".localized, index + 1, items.count)
+
+            // in case pillbuttonbar is used as .tabbar, adding our own index would be repetitive
+            if self.accessibilityTraits != .tabBar {
```

Comment:
I think this may still need to be gated by iOS version checks. Seems like `.tabBar` didn't add the index info from for iOS versions `14.0 ≤ version < 14.6`, based on other PRs? So we'd still need to add it unconditionally in those cases.